tests/e2e/smoketest-with-jaeger/01-assert.yaml

```diff
@@ -161,6 +161,10 @@ spec:
       protocol: TCP
       targetPort: http-memberlist
   selector:
+    app.kubernetes.io/created-by: tempo-controller
+    app.kubernetes.io/instance: simplest
```

Comment:
Couldnt that lead to a conflict with other applications like a jaeger instance called `simplest`? I dont know the convention for `app.kubernetes.io/instance`. ^^

Reply:
all labels need to match, so it would only create a conflict if the jaeger instance also has all 5 labels set:
```
app.kubernetes.io/created-by: tempo-controller
app.kubernetes.io/instance: simplest
app.kubernetes.io/managed-by: tempo-controller
app.kubernetes.io/name: tempo
tempo-gossip-member: "true"
```
afaics this should never be the case?

Reply:
right, but if i would filter for instance=simplest in another application/cli tool or eq this could be the case. To make this clear, ive not strong opinion on that. ^^ Its more a stupid question. :smile: 
![image](https://user-images.githubusercontent.com/10403402/203991202-93ceea12-c73a-4a1e-9b5e-885d72729a62.png)
Found that thing here: https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels

Reply:
I peeked at the official tempo helm chart, and they're using the following selector for the gossip service:
```
selector:
  app.kubernetes.io/instance: my-release
  app.kubernetes.io/name: tempo
  app.kubernetes.io/part-of: memberlist
```

and pods have the following labels (e.g. the distributor pod):
```
labels:
  app.kubernetes.io/component: distributor
  app.kubernetes.io/instance: my-release
  app.kubernetes.io/managed-by: Helm
  app.kubernetes.io/name: tempo
  app.kubernetes.io/part-of: memberlist
  app.kubernetes.io/version: 1.5.0
  helm.sh/chart: tempo-distributed-0.27.9
  pod-template-hash: 79c5d88cd4
```

Reply:
thanks :)